mdoc-sbt/src/main/scala/mdoc/MdocPlugin.scala

```diff
@@ -154,7 +160,14 @@ object MdocPlugin extends AutoPlugin {
 
           val linkerDependency = binaryVersion match {
             case "3" => "org.scala-js" % "scalajs-linker_2.13" % sjsVersion
-            case other => "org.scala-js" % s"scalajs-linker_$other" % sjsVersion
+            case scalaBinaryVersion =>
```

Comment:
is this purely cosmetic? if yes, please undo

Reply:
The name change was proposed by Tomasz and I believe is indeed more descriptive. If you confirm a strong preference to change it back I will